x-pack/plugins/spaces/public/space_list/space_list_internal.tsx

```diff
@@ -76,7 +78,7 @@ export const SpaceListInternal = ({
           defaultMessage: `* All spaces`,
         }),
         initials: '*',
-        color: '#D3DAE6',
+        color: euiTheme.colors.vis.euiColorVis3,
```

Comment:
@SiddharthMantri can you please provide us with a screenshot of this? cc @gvnmagni

Reply:
@andreadelrio Of course! i just added the before and after to the description above.

Reply:
@SiddharthMantri Thanks, it looks like the color before was this gray though (#D3DAE6), not the purple shown in your screenshot? 
<img width="261" alt="image" src="https://github.com/user-attachments/assets/71394c47-e5b1-42e7-8e9f-ba9b68debb9a">

Reply:
@andreadelrio Oops, you're right. I was using the recommendation from here: https://github.com/elastic/kibana/pull/201795#issuecomment-2505591515 Happy to switch to whichever color is closest.

Reply:
@SiddharthMantri thanks! I asked Lene a clarifying question regarding this.